test/Test/Database/LSMTree/Internal/Snapshot/Codec/Golden.hs

```diff
@@ -266,6 +361,14 @@ enumerateDiskCachePolicy =
 enumerateRunNumbers :: RunNumber
 enumerateRunNumbers = RunNumber magicNumber2
 
+enumerateOpenRunInfo :: SnapshotRun
+enumerateOpenRunInfo =
+    SnapshotRun {
+      snapRunNumber  = enumerateRunNumbers,
+      snapRunCaching = CacheRunData,
+      snapRunIndex   = Compact
+    }
```

Comment:
Should we maybe properly enumerate `snapRunCaching` and `snapRunIndex`?

Reply:
Having multiple values of this will mean yet more golden test data files.

Reply:
I've added:
```
--TODO: use a proper enumeration, but don't cause a combinatorial explosion of
-- golden tests. Perhaps do all combos as a direct golden test, but then where
-- it is embedded, just use one combo.
```